api/collections/permissions.py

```diff
@@ -113,3 +136,46 @@ def has_object_permission(self, request, view, obj):
                 has_pointer_auth = False
                 break
         return has_pointer_auth
+
+
+class OnlyAdminCanCreateDestroyCollectionSubmissionAction(permissions.BasePermission):
```

Comment:
Again, this is a confusingly-verbose permission name for something that is used in one place